include/revng/ABI/Definition.h

```diff
@@ -284,6 +284,16 @@ type: struct
     type: bool
     optional: true
 
+  - name: ReturnValueLocationIsReturned
+    doc: |
+      Specifies whether functions accepting a return value location should
+      return it back.
```

Comment:
Specifies whether functions whose return value is saved in a memory area pointed by an argument passed by the caller (SPTAR) should also return such pointer on `ReturnValueLocationRegister`/`ReturnValueLocationOnStack`.

Reply:
I went with
```cpp
      Specifies whether functions whose return value is saved in a memory area
      pointed by an argument passed by the caller (SPTAR) should also return
      such pointer using the first return value register.
```